mlkem/poly.c

```diff
@@ -494,23 +494,20 @@ void poly_basemul_montgomery_cached(poly *r, const poly *a, const poly *b,
     }
 }
 
-/*************************************************
- * Name:        poly_tomont
- *
- * Description: Inplace conversion of all coefficients of a polynomial
- *              from normal domain to Montgomery domain
- *
- *              Bounds: Output < q in absolute value.
- *
- * Arguments:   - poly *r: pointer to input/output polynomial
- **************************************************/
 #if !defined(MLKEM_USE_NATIVE_POLY_TOMONT)
 void poly_tomont(poly *r) {
-  unsigned int i;
+  int i;
   const int16_t f = (1ULL << 32) % MLKEM_Q;  // 1353
```

Comment:
Are we happy that this expression is "static" that it will be evaluated at compile time at all optimization levels by all compilers, and therefore in constant time?

Reply:
This is a public constant, so even if it isn't, it doesn't matter?

Reply:
Ah yes - no risk there.